test/OverlayTriggerSpec.js

```diff
@@ -111,6 +111,33 @@ describe('<OverlayTrigger>', () => {
     wrapper.assertSingle('div.test');
   });
 
+  it('Should show after mouseover trigger', (done) => {
+    const clock = sinon.useFakeTimers();
+
+    try {
+      const wrapper = mount(
+        <OverlayTrigger overlay={<Div className="test" />}>
+          <span>hover me</span>
+        </OverlayTrigger>,
+      );
+
+      wrapper.assertNone('.test');
+
+      wrapper.find('span').simulate('mouseover');
+
+      wrapper.assertSingle('div.test');
+
+      wrapper.find('span').simulate('mouseout');
+
+      clock.tick(50);
```

Comment:
after the mouseout we have to let the clock tick in order for the overlay to disappear (there is a bit of an animation w/setTimeout I think)